### ISSUE TITLE
The remote package repository is outdated

### ISSUE THREAD

**Issue by Ishtiyaq2402** [MiKTeX/miktex-packaging#261]
I have downloaded MikTex today (28/Nov/2021, 15:00 IST). I am unable to install. It is displaying "The remote package repository is outdated. You have to choose another repository.". Please help me, what to do?

Thanks

**Comment by edocevoli** [comment]
Thank you, it should work now. Please try again.

**Comment by Ishtiyaq2402** [comment]
Thank you. So, what should I do now? Should I again download and then install or I can directly install from the already downloaded distribution?

Thanks again

**Comment by Ishtiyaq2402** [comment]
Thank you. So, what should I do now? Should I again download and then install or I can directly install from the already downloaded distribution?

I tried to install from the previously downloaded version - setup-4.2-x64, that I downloaded today. But it is giving the error "The operation could not be completed for the following reasons: Timeout was reached Details: code="28" url=https://api2.miktek.org/hello". What to do?

Thanks again
